rfcs/0013-deprecate-currency-factory.md

```diff
@@ -0,0 +1,210 @@
+# Depreciation of Currency Factory - Assessment RFC
+
+This assessment aims to gather all relevant details and consequences of 
+depreciating Currency Factory and its replacements.
+
+## Reasoning
+
+There are several reasons we stand to benefit from replacing Currency Factory.
+These reasons are detailed below:
+
+### Currency Factory Ranges are Confusing
+  
+The Currency Factory ranges are a confusing abstraction that we don't 
+stand to gain from. We really only have two types of currencies native 
+(mintable) and non-native (external). However, we currently have many more 
+ranges that enforce artificial limitations on various abstractions of these two 
+currency types. This complicates the currency creation process and leads to 
+confusion while providing no benefit to us. 
+Since asset IDs aren't intended for human consumption, there is no requirement for them to be "easily consumable" - the end user should be concerned with the asset's _symbol_, not it's id.
+
+### No Permissionless Asset Creation
+
+> We do not have permissionless asset (non-payable/no sufficient) - it is 
```

Comment:
I'm don't really get this statement, why do we need to create new assets for governance and airdrops?
And what is meant by "no marketable assets"?

Reply:
If @dzmitry-lahoda could elaborate on this, that would be appreciated